features/old/cms/groups.feature

```diff
@@ -20,6 +20,7 @@ Feature: CMS groups
        And I press "Create"
       Then I should be on the groups page
        And I should see "le java enterprisers"
+       And I should see "nothing-to-see-here"
```

Comment:
Ouch, so there was a bug in the test that masked the bug in the code :smiling_face_with_tear:

Reply:
Classic